<?xml version="1.0" ?>
<sdf version="1.4">
  <model name="drill_blue_small">
    <pose>0 0 0.0505 0 0 0</pose>
    <static>false</static>
    <link name="body">
      <inertial>
        <pose>0.000001 -0.008431 -0.003743 0 0 0</pose>
        <mass>1</mass>
        <inertia>
          <ixx>0.38900e-3</ixx>
          <ixy>-0.00001e-3</ixy>
          <ixz>-0.00002e-3</ixz>
          <iyy>0.32198e-3</iyy>
          <iyz>-0.01703e-3</iyz>
          <izz>0.40459e-3</izz>
        </inertia>
      </inertial>
      <visual name="visual">
        <cast_shadows>true</cast_shadows>
        <geometry>
         <mesh>
           <uri>model://mug/meshes/mug.dae</uri>
         </mesh>
        </geometry>
      </visual>
      <collision name="collision">
        <surface>
          <friction>
            <ode>
              <mu>0.3</mu>
              <mu2>0.3</mu2>
            </ode>
          </friction>
        </surface>
        <geometry>
         <mesh>
           <uri>model://mug/meshes/mug.stl</uri>
         </mesh>
        </geometry>
      </collision>
    </link>
  </model>
</sdf>
